language/German/strings.po

```diff
@@ -3184,7 +3217,7 @@ msgstr "System ist ein Klon von\t%1$s\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2888
 msgid "System is Parent\t\n"
-msgstr ""
+msgstr "System ist übergeordnetes System\t\n"
```

Comment:
This is another case where terminology for “parent” has been made inconsistent with the filter panel.

Reply:
Fixed in the other places to be consistent